### ISSUE TITLE
Users unable to upload images in Forum

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#663]
New site.

A user (login name: Ginacross) has reported that they cannot upload an image to their forum post. The dialogue box that normally shows "upload, Web and Library" tabs does not show the "upload" tab.

I have also seen a few other posts from users that they have not been able to figure out how to upload pictures to the forum. 

This may be a simple permissions thing as when I log in I get all three tabs.

**Comment by djwesto** [comment]
Can you possibly add a screengrab showing what you think the issue is?

**Comment by GilesGibson** [comment]
![image](https://cloud.githubusercontent.com/assets/6060985/9329437/78dc062e-45a9-11e5-8b9e-184dd82b7cc2.png)

**Comment by djwesto** [comment]
Whoops, looks like there's no upload option active on the system. That's a bit weird.
@pmackay was it always that way? Think it might be a permissions issue?

**Comment by pmackay** [comment]
Are some users able to save images?

**Comment by GilesGibson** [comment]
I dont think that anyone has managed to upload images as yet. There were some comments made via email that they were struggling to do it. Some posts also had comments that they wanted to add a picture of their sofa/cat/car etc that they wanted to give away but couldn't get the image up.

**Comment by GilesGibson** [comment]
By the way, I do get the tab so have been telling people to just use the upload option.... that they haven't got. I think they just gave up and posted text only.

**Comment by djwesto** [comment]
Definitely sounds like a permissions issue then.

**Comment by djwesto** [comment]
Is it this, @pmackay ?
![image](https://cloud.githubusercontent.com/assets/702685/9355520/02180fdc-4672-11e5-818d-0b843724d8d9.png)

**Comment by pmackay** [comment]
Yes probably :) Want to turn on for auth users?

**Comment by djwesto** [comment]
Well, yes but presumably all user groups would want it wouldn't they? Does the permission cascade upwards?

**Comment by pmackay** [comment]
Authenticated users means any user that is logged in, irrespective if they
are member of group or not.

**Comment by pmackay** [comment]
Have enabled permission for this, please test and verify.

**Comment by GilesGibson** [comment]
I have asked the user to check
